bundles/org.openhab.binding.panamaxfurman/src/main/java/org/openhab/binding/panamaxfurman/internal/protocol/PowerConditionerChannel.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.panamaxfurman.internal.protocol;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.panamaxfurman.internal.PanamaxFurmanAbstractHandler;
+import org.openhab.binding.panamaxfurman.internal.PanamaxFurmanConstants;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Enum for channel that is exposed by the device
+ *
+ * @author Dave Badia - Initial contribution
+ */
+@NonNullByDefault
+public enum PowerConditionerChannel {
+    POWER("power", OnOffType.class, true),
+    ID("id", null, false),;
+
+    private final static Logger logger = LoggerFactory.getLogger(PowerConditionerChannel.class);
+
+    private final static Map<String, PowerConditionerChannel> CHANNEL_STRING_TO_ENUM_TABLE = new ConcurrentHashMap<>();
+
+    private final String name;
+    private final @Nullable Class<? extends State> stateClass;
+    private final boolean outletSpecific;
+
+    private PowerConditionerChannel(String name, @Nullable Class<? extends State> stateClass, boolean outletSpecific) {
+        this.name = name;
+        this.stateClass = stateClass;
+        this.outletSpecific = outletSpecific;
+    }
+
+    public String getChannelName(@Nullable Integer outletNumber) {
+        if (outletNumber == null) {
+            return getName();
+        }
+        if (isOutletSpecific()) {
+            return PanamaxFurmanAbstractHandler.getChannelUID(name, outletNumber);
+        } else {
+            logger.warn("{} was called with outlet #{} but is not outlet specific. ()", this, outletNumber);
```

Comment:
```suggestion
            logger.debug("{} was called with outlet #{} but is not outlet specific. ()", this, outletNumber);
```

Reply:
This should be reported if it occurs

Reply:
Is this a configuration mistake? If so, you could change the message a bit to give the user a hint. For example:
'Non-specific {} called with outlet #{}, check your configuration. ()'